src/signals/IncidentMap/components/FilterPanel/FilterPanel.tsx

```diff
@@ -64,7 +59,7 @@ export const FilterPanel = ({ filters, setFilters, setMapMessage }: Props) => {
   }
 
   return (
-    <StyledPanelContent data-testid="filterCategoryPanel">
+    <Fragment>
```

Comment:
fragments can be replaced by <> which is more dense, and you don't need to include Fragment for it!

Reply:
Guess it's a taste difference. I prefer to use Fragment instead of <> because just a <> can me missed very easily while reading code. But i'll replace it. 
FYI: apparently the difference is that Fragment can take a key attribute ([docs](https://medium.com/fasal-engineering/what-are-react-fragments-or-the-react-empty-tags-190253582905))